cost/aws/savings_plan/expiration/README.md

```diff
@@ -0,0 +1,44 @@
+# AWS Expiring Savings Plans
+
+## What it does
+
+This Policy Template leverages the Savings Plan API for savings plan information. It will notify only if expiration is within the timeframe specified in `Number of days to prior to expiration date to trigger incident` field. It will email the user specified in `Email addresses of the recipients you wish to notify`.
+
+## Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+- *Number of days to prior to expiration date to trigger incident* - enter the number of days you want before the Reserved Instance expires.
```

Comment:
think this should say "Enter the number of days before the Reserved Instance expires that you want to be notified on."

Reply:
Also maybe worth rewording this so it is explicit that this is for Savings Plans rather than RIs